modules/apps/portal-url-builder/portal-url-builder-api/src/main/java/com/liferay/portal/url/builder/AbsolutePortalURLBuilder.java

```diff
@@ -68,6 +68,12 @@
 	 * web context path.
 	 *
 	 * <p>
+	 * If the requested module resource is a JavaScript file or a stylesheet,
+	 * better use {@link #forModuleScript(Bundle, String)} or {@link
+	 * #forModuleStylesheet(Bundle, String)} instead.
+	 * </p>
+	 *
```

Comment:
Can you add @review. Brian let's you add javadocs freely if they know that they are going to be reviewed in the future. I don't know about the <p> tags though

Reply:
Done :+1: